org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/handlers/GenerateToStringActionTest.java

```diff
@@ -177,4 +177,23 @@ public void testGenerateToStringQuickAssist_emptyFields() throws JavaModelExcept
 		quickAssistActions = CodeActionHandlerTest.findActions(codeActions, JavaCodeActionKind.QUICK_ASSIST);
 		Assert.assertFalse(CodeActionHandlerTest.commandExists(quickAssistActions, CodeActionHandler.COMMAND_ID_APPLY_EDIT, ActionMessages.GenerateToStringAction_label));
 	}
+
+	@Test
+	public void testGenerateToStringQuickAssistExists() throws JavaModelException {

```

Comment:
For this method and the one below. Instead of calling them `testGenerateToStringQuickAssistExists`, `testHashCodeEqualsQuickAssistExists` , I would call them ` testNoGenerateToStringQuickAssist` & `testNoHashCodeEqualsQuickAssist` .

Basically, the current name implies that the quick-assist should exist, but I know what you mean is that the method exists so the quick-assist shouldn't. It just might be confusing with the current name.

Reply:
Yeah, Let's make them clear.